barretenberg/cpp/src/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 
 if(CMAKE_CXX_COMPILER_ID MATCHES "GNU")
-    add_compile_options(-fconstexpr-ops-limit=100000000)
+    add_compile_options(-fconstexpr-ops-limit=100000000 -Wno-psabi)
```

Comment:
This was confusing people looking at GCC. We don't need to be warned about the ABI change